### PR TITLE
Change pixels per point on mac to be 2 instead of 1

### PR DESCRIPTION
**Pull request by OlshaMB**

### CODE REVIEW

**Comment by Umatriz** [comment]
Making it a setting sound as a really great idea. I would appriciate if you could do it,

**Comment by OlshaMB** [comment]
Where should I put this setting?
Putting this in the [`Settings`](https://github.com/Umatriz/nomi/blob/main/crates/nomi-core/src/configs/user.rs) doesn't make sense

**Comment by OlshaMB** [comment]
Also shouldn't nomi-core be stateless(at least storing only storage paths)?

**Comment by Umatriz** [comment]
> Where should I put this setting? Putting this in the [`Settings`](https://github.com/Umatriz/nomi/blob/main/crates/nomi-core/src/configs/user.rs) doesn't make sense

Make something like `ClientSettings` and put it in the `client` crate

**Comment by Umatriz** [comment]
> Also shouldn't nomi-core be stateless(at least storing only storage paths)?

I don't understand what do you mean. You may create an issue because it seems to be out of scope for this PR.

File: crates/client/src/context.rs [hunk 1]
```diff
@@ -24,6 +25,13 @@ use std::{
     sync::mpsc::{Receiver, Sender},
 };
 
+fn default_pixels_per_point_value() -> f32 {
+    #[cfg(not(target_os = "macos"))]
+    return 1.2;
+    #[cfg(target_os = "macos")]
+    2.0
+}
+
```
Comment (Umatriz) [inline crates/client/src/context.rs]: Consider using [`cfg!`](https://doc.rust-lang.org/std/macro.cfg.html) macro

File: crates/client/src/context.rs [hunk 2]
```diff
@@ -56,6 +66,11 @@ impl AppContext {
         let (settings_java_buf_tx, settings_java_buf_rx) = std::sync::mpsc::channel();
         let profiles =
             read_toml_config_sync::<VersionProfilesConfig>("./.nomi/configs/Profiles.toml");
+        let client_settings_res =
+            read_toml_config_sync::<ClientSettings>("./.nomi/configs/ClientSettings.gui.toml");
+        let client_settings = client_settings_res.unwrap_or(ClientSettings {
+            pixels_per_point: Some(default_pixels_per_point_value()),
```
Comment (Umatriz) [inline crates/client/src/context.rs]: Implement `Default`for `ClientSettings` and use it

File: crates/client/src/context.rs [hunk 3]
```diff
@@ -99,10 +115,18 @@ impl AppContext {
...
             settings_username: Default::default(),
+            settings_pixels_per_point: client_settings
+                .pixels_per_point
+                .unwrap_or(default_pixels_per_point_value()),
         })
     }
 
     pub fn show_main(&mut self, ui: &mut Ui) {
+        ui.ctx().set_pixels_per_point(
+            self.client_settings
```
Comment (Umatriz) [inline crates/client/src/context.rs:118]: Maybe use `self.settings_pixels_per_point` there?

**Review by Umatriz — CHANGES_REQUESTED** [review]

File: crates/client/src/context.rs (hunk 1 above)
Comment (OlshaMB) [reply crates/client/src/context.rs:34]: This is not comptime

File: crates/client/src/context.rs (hunk 3 above)
Comment (OlshaMB) [reply crates/client/src/context.rs:118]: This makes UI resizing live and janky

**Pull request merged**